ci/scripts/git2changelog

```diff
@@ -0,0 +1,188 @@
+#!/bin/bash
```

Comment:
Did you consider to use ready to use tools like `git-dch`?

https://manpages.debian.org/jessie/git-buildpackage/git-dch.1.en.html